experimental/DrinfeldHeckeAlgebras/src/DrinfeldHeckeAlgebra.jl

```diff
@@ -0,0 +1,669 @@
+#######################################
+# Struct and methods to generate and handle concrete or generic Drinfeld-Hecke algebras
+#
+# Cassandra Koenen, 2025
+#######################################
+
+#######################################
+# Struct and different constructors for Drinfeld-Hecke algebras
+#######################################
+
+@doc raw"""
+    DrinfeldHeckeAlgebra{T <: FieldElem, S <: RingElem} <: NCRing
+
+Type for representing a Drinfeld-Hecke algebra for a group $G$. The parameter ```T``` is the element type over which
+$G$ is defined, ```S``` is the ring element type over which the associated Drinfeld-Hecke form is defined.
+"""
+mutable struct DrinfeldHeckeAlgebra{T <: FieldElem, S <: RingElem} <: NCRing
+  form::DrinfeldHeckeForm{T}
+  
+  # Create from forms input
+  function DrinfeldHeckeAlgebra(forms::Dict)
+    κ = DrinfeldHeckeForm(forms)
```

Comment:
per [the Oscar styleguide](https://docs.oscar-system.org/stable/DeveloperDocumentation/styleguide/#Unicode), unicode should not be used in code